.github/workflows/dev-aws-ecr.yml

```diff
@@ -57,7 +57,6 @@ jobs:
         CERT: ${{ secrets.SAML_CERT }}
 
       run: |
-        sudo echo $CERT > /usr/local/share/ca-certificates/federation.dev.cce.af.mil.crt
```

Comment:
Why isn't this needed in the C1 step? I think I'm confused as to how the docker image differs between this step and the AWS step

Reply:
Federation use used for SAML endpoint. C1 Artifactory doesn't have anything to do with SAML so this cert isn't necessary. While the C1 step does not need this cert to access Artifactory, it does still need `saml.pem` to exist so it can build the container. Fixing this.